### PR TITLE
Feature/soul upgrade

### PR DESCRIPTION
**Pull request by johnc303**
Renamed all HTML template files to be hbs templates for better syntax highlighting.
Adjusted to use value instead of current to tie into pre-existing helpers.
Added wide block title class.
Moved soulpoint & XP into own stylesheet.
Styled new soulpoint & flare container (WIP).
Added max soul points value.
Added logic to mark when soulpoints are over maximum.
Added new perk bonus type to increase said total.
Fixed bug whereby some JS files were still referencing the old .html extenstions rather than the new .hbs ones.

### CODE REVIEW

File: module/actor/actor.js [hunk 1]
```diff
@@ -567,10 +555,38 @@ export class PrimePCActor extends Actor
         return totalCost;
     }
 
+    updateSoulAndXP(actorSystemData)
+    {
+        const primeCost = this.getTotalCost(actorSystemData.primes);
+        const perkSoulCost = this.getTotalPerkCost("perkCostSoul");
```
Comment (biz812) [inline module/actor/actor.js]: The use of perkCostSoul and perkSoulCost interchangeably feels like a bit of a gotcha, any way to consolidate these to one or the other?

File: module/actor/actor.js [hunk 2]
```diff
@@ -567,10 +555,38 @@ export class PrimePCActor extends Actor
...
+
+        const soulPointBonusToMax = this.getStatBonusesFromItems("soul.max.value");
+        actorSystemData.soul.max.value = actorSystemData.soul.max.base + soulPointBonusToMax;
+        if (actorSystemData.soul.max.value > actorSystemData.soul.max.absolute)
+        {
+            actorSystemData.soul.max.value = actorSystemData.soul.max.absolute;
+        }
+
+        const refinementCost = this.getTotalCost(actorSystemData.refinements);
+        const perkXPCost = this.getTotalPerkCost("perkCostXP");
```
Comment (biz812) [inline module/actor/actor.js]: The same here perkCostXP Vs perkXPCost

File: module/actor/actor.js (hunk 1 above)
Comment (johnc303) [reply module/actor/actor.js]: Good shout, have standardised them

File: module/actor/actor.js (hunk 2 above)
Comment (johnc303) [reply module/actor/actor.js]: Good shout, have standardised them

**Comment by biz812** [comment]
Looks good to me 👍

**Pull request merged**